components/Main/NewsletterSignUp/SignUpForm/SignUpForm.module.css

```diff
@@ -0,0 +1,18 @@
+.form > * {
```

Comment:
Not sure about the universal child selector here. What is the intent and could we be more specific?

Like could classes be added for the elements that need these styles?

Reply:
Thanks for the advice to avoid `*` selector. I realize that by using the specific class or tag selector, it would be easier to understand what style does the element has now.